src/main/java/org/apache/maven/plugin/resources/remote/AbstractProcessRemoteResourcesMojo.java

```diff
@@ -605,7 +613,7 @@ protected Map<Organization, List<MavenProject>> getProjectsSortedByOrganization(
         return organizations;
     }
 
-    protected boolean copyResourceIfExists(File file, String relFileName, VelocityContext context)
+    protected boolean copyResourceIfExists(File outputFile, String relFileName, VelocityContext context)
```

Comment:
relFileName, what is this? Is rel "relative" or release? If relative, how is a file name relative? Please rename for clarity.

Reply:
it is - `bundleResourceName`